plugin/src/main/resources/metadata/messages.xml

```diff
@@ -981,6 +981,76 @@ uses. <b>PBKDF2</b> should be use to hash password for example.</p>
     <BugCode abbrev="SECSHA1">SHA-1 is a weak hash function</BugCode>
 
 
+    <!-- Weak SSL/TLS protocols -->
+    <Detector class="com.h3xstream.findsecbugs.crypto.WeakTLSDetector">
+        <Details>Identify the usage of weak SSL/TLS protocols</Details>
+    </Detector>
+
+    <BugPattern type="DEFAULT_HTTP_CLIENT">
+        <ShortDescription>DefaultHttpClient with default constructor is not compatible with TLS 1.2</ShortDescription>
+        <LongDescription>DefaultHttpClient with default constructor is not compatible with TLS 1.2</LongDescription>
+        <Details>
+            <![CDATA[
+<p>
+    <b>Vulnerable Code:</b><br/>
+<pre>
+    HttpClient client = new DefaultHttpClient();
+</pre>
+</p>
+<p>
+<p><b>Solution:</b><br/>
+
+Upgrade your implementation to use one of the recommended constructs and configure https.protocols JVM option to include TLSv1.2:</p>
```

Comment:
yes SystemDefaultHttpClient is a valid implementation. Added sample solutions in the messages.